packages/block-library/src/cover/test/__snapshots__/edit.native.js.snap

```diff
@@ -33,72 +33,72 @@ exports[`color settings toggles between solid colors and gradients 1`] = `
 `;
 
 exports[`minimum height settings changes the height value between units 1`] = `
-"<!-- wp:cover {"url":"https://cldup.com/cXyG__fTLN.jpg","id":10710,"dimRatio":50,"overlayColor":"foreground","isUserOverlayColor":true,"minHeight":50,"minHeightUnit":"px","isDark":false} -->
-<div class="wp-block-cover is-light" style="min-height:50px"><span aria-hidden="true" class="wp-block-cover__background has-foreground-background-color has-background-dim"></span><img class="wp-block-cover__image-background wp-image-10710" alt="" src="https://cldup.com/cXyG__fTLN.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","placeholder":"Write title…","fontSize":"large"} -->
+"<!-- wp:cover {"url":"https://cldup.com/cXyG__fTLN.jpg","id":10710,"dimRatio":50,"overlayColor":"foreground","minHeight":50,"minHeightUnit":"px","isDark":false} -->
```

Comment:
How'd you generate these updates? 

I'm wondering why the `isUserOverlayColor` attribute was removed.

Reply:
I used `npm run test:native:update` to update the snapshots.